Modules/zlibmodule.c

```diff
@@ -1452,12 +1450,12 @@ zlib_crc32_impl(PyObject *module, Py_buffer *data, unsigned int value)
             buf += (size_t) UINT_MAX;
             len -= (size_t) UINT_MAX;
         }
-        signed_val = crc32(value, buf, (unsigned int)len);
+        value = crc32(value, buf, (unsigned int)len);
         Py_END_ALLOW_THREADS
```

Comment:
The signature of `crc32()` is `uLong crc32(uLong crc, const Bytef *buf, uInt len)`